docs/examples/ensemble-window.ipynb

```diff
@@ -0,0 +1,429 @@
+    "\n",
+    "    rand_indexes[cost] = np.around(randindex(bkps, predicted_bkps), 3)\n",
+    "    scores.append(algo.score)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have the performance of the classical method on each cost function, let's dive into the ensemble method. We will then compare the results."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "qB3Nhz3njORl"
+   },
+   "source": [
+    "### Ensemble method: multiple costs for one prediction\n",
+    "\n",
+    "The paper we rely on [Katser2021](#Katser2021), uses the score from each cost function to detect the changepoints. This is why we stored them in the previous section."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "!!! note\n",
+    "    As a reminder, [here](../user-guide/detection/window.md) is how the score is computed for the window-based search method, it can also be called the *discrepancy measure*."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's see how those scores look like. In the following cell, each row represents the score for a specific cost function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 729
+    },
+    "id": "nBk9sihcLk2d",
+    "outputId": "90bf9753-3ca1-43fd-a3c9-d103c5a923a0"
+   },
+   "outputs": [],
+   "source": [
+    "scores = np.array(scores).T\n",
+    "\n",
+    "# Display the scores with the change points to predict\n",
+    "append_scores = np.append(\n",
+    "    np.ones((window_size // 2, len(cost_functions))) * float(\"inf\"), scores, axis=0\n",
+    ")\n",
+    "_ = rpt.display(append_scores, bkps)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The idea of the CPD Ensemble method is to scale and aggregate the scores with some well chosen scaling and aggregation function to get new score which will allow us to get a better prediction of the changepoints.\n",
+    "\n",
+    "In the paper, they perform multiple tests on several scaling and aggregation functions on two datasets. In turned out that the *MinAbs* scaling function and the *WeightedSum* aggregation function worked the best.\n",
+    "\n",
+    "The *MinAbs* function is defined as follows:\n",
+    "For $s$ a timeseries, \n",
+    "\n",
+    "$$\n",
+    "\\textit{MinAbs}(s)_i = \\frac{s_i}{|\\min_{j}{s_j}|}\n",
+    "$$\n",
+    "\n",
+    "The *WeightedSum* function is defined as follows:\n",
+    "Let $s^n, n \\in \\{1, ..., N\\}$ be $N$ timeseries, we have to distinguish the \"original\" timeseries $s^n$ from its scaled version $\\overline{s}^n = \\textit{MinAbs}(s^n)$. We then have\n",
+    "\n",
+    "$$ \\textit{WeightedSum}((s_n)_{n \\in \\{1, ..., N\\}})_i = \\sum_{n \\in \\{1, ..., N\\}}{\\lambda_n \\overline{s}_i^n} \n",
```

Comment:
```suggestion
    "$$ \\textit{WeightedSum}((s_{i}^{n})_{n \\in \\{1, ..., N\\}}) = \\sum_{n \\in \\{1, ..., N\\}}{\\lambda_n \\overline{s}_i^n} \n",
```

Reply:
Same here

Reply:
I edited my suggestion in order to keep only the `s_n` --> `s^n` modification.